src/ansys/fluent/core/solver/flunits.py

```diff
@@ -59,13 +107,9 @@ def make_python_fl_unit_table(scheme_unit_table):
     "elec-charge": "A h",
```

Comment:
this is wrong. 
I'm going to add an extra constraint: Instantiate a unit quantity for each unit and compare its value to the SI value.

Reply:
This is all done and updated